gotechnica/platform-2020
# Finished adding in checklist function to finished project submission

# Description 
1. There is a checklist of items the hacker must complete to submit their project. Right now you can click the checkbox to check it off, but should be able to click the label next to the checkbox to check it off as well.
1. Added an @click prop that toggles whether the checkbox is checked and the item in the checklist is completed.

# Steps to test
1. cd into the frontend
1. run `npm run serve`
1. go to "Project Submission" page, make sure that you can click the text label and toggle whether the checkbox is checked
